### ISSUE TITLE
Fix "Rotor Callout" for the footer (all pages)

### ISSUE THREAD

**Issue by oddball-lindsay** [department-of-veterans-affairs/va.gov-team#98041]
This finding was flagged by accessibility for our `/review-and-submit` page, but they indicated that this was actually affecting all pages in our form experience.

## Expected vs. Actual
**Expected**: The footer should be labeled "footer" for VoiceOver Rotor users, ideally by using the footer tag.

**Actual**: Testing display of content on VoiceOver Rotor, noticed the footer was labeled as “content information” instead of “footer” under Landmarks (see screenshot).

## Steps to Reproduce

1. Use Voiceover, then open the rotor menu by pressing “caps lock + u”.
2. Navigate to the landmark screen by using left/right arrow keys, you can traverse the landmarks by using the up down arrow keys. 
3. Press the Enter key while on “content information” and it will highlight the first item on the footer as seen in screenshot below.

## Examples
![Image](https://github.com/user-attachments/assets/3b70a0cf-e924-49ba-899c-e09d6389ae5e)

## Tasks
- [ ] Using the footer semantic tag should fix this.
